public/app/features/backup/components/DescriptionBlock/DescriptionBlock.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { useStyles } from '@grafana/ui';
+import { DescriptionBlockProps } from './DescriptionBlock.types';
+import { Messages } from './DescriptionBlock.messages';
+import { getStyles } from './DescriptionBlock.styles';
+
+export const DescriptionBlock: FC<DescriptionBlockProps> = ({ description }) => {
+  const styles = useStyles(getStyles);
+
+  return (
+    <div className={styles.descriptionWrapper}>
+      <span>{Messages.description}</span>
+      <pre>{description}</pre>
+    </div>
```

Comment:
You should consider adding `data-qa` attributes here

Reply:
done 👍